hs_core/urls.py

```diff
@@ -73,15 +73,16 @@
     url(r'^_internal/group_membership/(?P<token>[-\w]+)/(?P<uidb36>[-\w]+)/(?P<membership_request_id>[0-9]+)/',
         views.group_membership,
         name='group_membership'),
+    url(r'^_internal/metadata_review/(?P<shortkey>[0-9a-f-]+)/\
+        (?P<action>[a-z]+)/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[-\w]+)/',
```

Comment:
Link to token generation:
https://github.com/django/django/blob/3.2/django/contrib/auth/tokens.py#L72-L85

Reply:
I originally thought it was my use of `uidb36` instead of 64 that was causing the authentication via link not to work on this endpoint.

Then I realized that it was simply the order of urls (`metadata_review` vs `metadata_review_noauth`) and the regex matching pattern.

But I figured it wouldn't hurt to leave the uidb64 modifications in here...